src/index.ts

```diff
@@ -9,35 +10,22 @@ const movieSearchBox = document.getElementById(
   'movie-search-box'
 ) as HTMLInputElement;
 const searchList = document.getElementById('search-list') as HTMLInputElement;
+let searchTerm = movieSearchBox.value.trim();
 
 // Get Search Term to call API
-function findMovies(): void {
-  let searchTerm = movieSearchBox.value.trim();
+function searchMovies(searchTerm: string): void {
   if (searchTerm.length > 0) {
-    searchList.classList.remove('hide-search-list');
-    loadMovies(searchTerm);
-  } else {
-    searchList.classList.add('hide-search-list');
+    fetchMovieList();
```

Comment:
this is a redundant call. You're already calling `fetchMovieList` in the `loadMovies` on the next line. I caught this because `fetchMovieList` returns `Promise<Movie_Search>` and this line isn't using any return. A great example of how TypeScript helps with collaboration/maintenance/scalability of a codebase!

Reply:
Your right i resolved this